pyEpiabm/pyEpiabm/core/cell.py

```diff
@@ -96,25 +98,26 @@ def notify_person_status_change(
             Person's old infection status
         new_status : InfectionStatus
             Person's new infection status
-
+        age_group : Age group index
```

Comment:
good idea

Reply:
I've realised that we don't actually pass the person to that function, just some explicit attributes. (As indeed, we do in the notify person status change, the other interaction with the counter). I don't think it would be any easier to pass the person in as an argument instead of just their age, and feels like that would require most info being passed around.

Reply:
That makes sense - I was just worried about the duplication of passing both but very happy with you not passing the person here if it is not needed!